bgpd/bgp_mplsvpn.c

```diff
@@ -460,99 +460,104 @@ prefix_rd2str (struct prefix_rd *prd, char *buf, size_t size)
 /* For testing purpose, static route of MPLS-VPN. */
 DEFUN (vpnv4_network,
        vpnv4_network_cmd,
-       "network A.B.C.D/M rd ASN:nn_or_IP-address:nn tag WORD",
+       "network A.B.C.D/M rd ASN:nn_or_IP-address:nn <tag|label> (0-1048575)",
```

Comment:
this one and the two commands below could be compressed into a single command as
`network A.B.C.D/M rd ASN:nn_or_IP-address:nn [<tag|label> (0-1048575) [route-map WORD]]`